lunaria.config.json

```
@@ -0,0 +1,57 @@
+{
+	"$schema": "./node_modules/@lunariajs/core/config.schema.json",
+	"repository": {
+		"name": "solidjs/solid-docs-next",
+		"branch": "main",
+		"hosting": "github"
+	},
+	"defaultLocale": {
+		"label": "English",
+		"lang": "en"
+	},
+	"files": [
+		{
+			"type": "universal",
+			"location": "src/routes/**/*.{mdx,md,tsx}",
+			"pattern": "src/routes/@lang/@path"
+		},
+		{
+			"location": "src/i18n/dictionaries/**/ui.ts",
+			"pattern": "src/i18n/dictionaries/@lang/@path",
+			"type": "dictionary"
+		}
+	],
+	"locales": [
+		{
+			"label": "Português do Brasil",
+			"lang": "pt-br"
+		}
+	],
+	"outDir": "./public/i18n-status",
+	"dashboard": {
+		"site": "https://docs.solidjs.com/i18n-status",
+		"title": "Solid Docs Localization Status",
+		"description": "Translation progress tracker for the Solid Docs site",
+
```

Since the file paths for the content here are very extensive, we could use the [`dashboard.basesToHide`](https://lunaria.dev/reference/configuration/#dashboardbasestohide) option to make it easier to scam while reading the dashboard later. Totally open to what you think works best!
```suggestion
    "basesToHide": ["src/routes/docs/", "src/i18n/dictionaries/"],
```